pyphare/pyphare/pharesee/hierarchy/__init__.py

```
@@ -2,16 +2,20 @@
 from .vectorfield import VectorField
 from .hierarchy import PatchHierarchy
 from pyphare.core.phare_utilities import listify
+from . import func
 
-__all__ = [
-    "ScalarField",
-    "VectorField",
-    "PatchHierarchy",
-]
+__all__ = ["ScalarField", "VectorField", "PatchHierarchy", "func"]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)